makefile

```diff
@@ -1,9 +1,9 @@
 # Documentation: https://www.gnu.org/software/make/manual/html_node/index.html
-VERSION=0.0.0.`git rev-parse --short HEAD`
+VERSION=0.2.0.`git rev-parse --short HEAD`
 GOFLAGS=-i -v -ldflags "-s -w -X main.Version=${VERSION}" -installsuffix cgo
 
-prowler.1m.cgo: prowler.go
-	go build ${GOFLAGS} -o prowler.1m.cgo prowler.go
+prowler.1m.cgo: prowler2.go
+	go build ${GOFLAGS} -o prowler.1m.cgo prowler2.go
```

Comment:
I will change this to spit out both clients; the existing Prowler binary and the Prowler2 binary